docker/wmagent/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN adduser -u 31961 -g 1399 cmst1
 RUN install -o cmst1 -g 1399 -d /data/srv /data/admin /data/certs
 RUN chown root:1399 /data
 
+ARG WMA_TAG=None
```

Comment:
Where does `WMA_TAG` come from? Shouldn't this be using the ENV keyword instead of ARG?